server/todo-api/controllers/controller.ts

```
@@ -121,5 +121,30 @@ export class Controller {
       next(bodyErrors);
     }
   }
+
+
+  async getAll(req: Request, res: Response, next: NextFunction): Promise<void> {
+    const { createdBy } = req.body;
+
+    if (isObjectId(createdBy)) {
```

Deberiamos aplicar la estrategia de fail fast para hacer mas entendible el codigo. seria mejor
```typescript
if(!isObjectId(createdBy)){
   const error: HTTPError = {
        status: 400,
        name: 'Bad request',
        message: `createdBy must by a ObjectId value but recived a ${typeof createdBy}`,
      };
     return next(error);
}`